pkg/gohai/filesystem/filesystem_windows.go

```diff
@@ -127,27 +126,27 @@ func getFileSystemInfo() (interface{}, error) {
 		moreData := true
 		for moreData {
 			outstring := convertWindowsString(buf)
-			sz, _ := getDiskSize(outstring)
-			var capacity string
-			if 0 == sz {
-				capacity = "Unknown"
-			} else {
-				capacity = strconv.FormatInt(int64(sz)/1024.0, 10)
+
+			size, _ := getDiskSize(outstring)
+			sizeKB := uint64(0)
+			if size != 0 {
+				sizeKB = uint64(size) / 1024
 			}
+
```

Comment:
We don't need the if, `0/1024` is 0.
```suggestion
			sizeKB := uint64(size) / 1024
```

Reply:
That's from the original PR too, really not sure why it was written like that
And it seems like the cast is useless, `size` is already a uint64

Reply:
Done in f7b0f05a0b2fd1909a2437e3e024a12f64edde34